ftrip.io.catalog-service/ftrip.io.catalog-service/Startup.cs

```diff
@@ -0,0 +1,74 @@
+using ftrip.io.framework.auth;
+using ftrip.io.framework.CQRS;
+using ftrip.io.framework.ExceptionHandling.Extensions;
+using ftrip.io.framework.Globalization;
+using ftrip.io.framework.Installers;
+using ftrip.io.framework.Mapping;
+using ftrip.io.framework.messaging.Installers;
+using ftrip.io.framework.Persistence.Sql.Mariadb;
+using ftrip.io.framework.Swagger;
+using ftrip.io.framework.Validation;
+using ftrip.io.framework.HealthCheck;
+using ftrip.io.catalog_service.Persistence;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace ftrip.io.catalog_service
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            InstallerCollection.With(
+                new SwaggerInstaller<Startup>(services),
+                new AutoMapperInstaller<Startup>(services),
+                new FluentValidationInstaller<Startup>(services),
+                new GlobalizationInstaller<Startup>(services),
+                new JwtAuthenticationInstaller(services),
```

Comment:
After updating to the latest version of the framework, please add 
> new EnviromentSecretsManagerInstaller(services),

above JwtAuthenticationInstaller. I added ISecretsManager interface from which we can read secrets. The reason for that is that I can unit test easier. Also, in future, we can switch implementations from reading env variables to using Azure secrets vault.